nautobot/ipam/filters.py

```diff
@@ -397,10 +393,29 @@ class IPAddressFilterSet(
         to_field_name="name",
         label="Namespace (name or ID)",
     )
+    has_interface_assignments = RelatedMembershipBooleanFilter(
+        field_name="interfaces",
+        method="_assigned_to_interface",
+        label="Has Interface Assignments",
+    )
+    ip_version = MultiValueNumberFilter()
```

Comment:
For some strange reason, `ip_version` is not mapped to `MultiValueNumberFilter`. Instead it is mapped to `MultipleChoiceFilter`. Maybe because its `models.IntegerField` has a `choices` attribute specified, so `django_filters's` default lookup mapped it to a `MultipleChoiceFilter`. We have to override it here to have the correct filter behavior.

Reply:
+1. It's an enum rather than a true number filter.